src/renderer/apps/multisend/def.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react'
+import {ReactComponent as MultisendIcon} from "./icon.svg"
+
+import { AppDefinition } from "../../components/app-definition"
+import MultiSendApp from "./multisend"
+import SvgIcon from '@material-ui/core/SvgIcon';
+
+const CVIcon = (props: {fontSize?: "default" | "large"}): JSX.Element => {
```

Comment:
nit: CVIcon was short for CelovoteIcon specifically, you might want to use a different name to avoid confusion.